cwms-servers/CWMS_HOME/bin/sendLogMessage

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+if [ ! -f $CWMS_EXE/bin/getProp ]
+then
+	echo " "
+	echo "Script \"$CWMS_EXE/bin/getProp\" was not found!"
+	echo "Unable to setup CWMS Enterprise Logging"
+	echo " "
+	exit 1
+fi
+
+LOGGING_HTTP_ENDPOINT=$(getProp cwms.enterprise.logging.http.url notFound cwms.properties $CWMS_HOME/config/properties)
+LOG_HOST=$(hostname)
+LOG_FACILITY='CWMS'
```

Comment:
why is this set to CWMS?

Reply:
well, the documentation no longer says since it's deprecated. Removed